src/RemoveObsoleteStatistics.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace DDB\Stats;
+
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Database\DatabaseManager;
+
+class RemoveObsoleteStatistics implements ShouldQueue
```

Comment:
What is this? An event or a job? If it's an event, it should be in DDB\Stats\Events, jobs in DDB\Stats\Jobs.

It's named like a job but dispatched as an event. But it's not an event, it's a job.

Jobs should use the following traits:

```php
use Illuminate\Bus\Queueable;
use Illuminate\Queue\SerializesModels;
use Illuminate\Queue\InteractsWithQueue;
```

Reply:
It's a [queued listener](https://laravel.com/docs/6.x/events#queued-event-listeners).

I move classes around a bit to make it more visible.